### DISCUSSION TITLE
Script to Bypass Zones

### DISCUSSION THREAD

**Discussion by meneerfrikkie** [Dilbert66/esphome-dsckeybus#118] [Q&A]
Hi there everyone, has anyone been able to come up with a script that bypasses set zones, it's summer right now and a couple of my window sensors are always open, so every time I have to bypass these zones before being ready to arm, any idea to automatically bypass? Much appreciated!

**Comment by Dilbert66** [comment]
Using the alarm_keypress service you can send the command "*199#" before your arming command to bypass all zones.   This might not work with older version of the dsc panels.  Alternatively, you can send *1xx# where xx is the zone you want to bypass.
